### ISSUE TITLE
CLI improvement: option to display only failed tests

### ISSUE THREAD

**Issue by yuriykuzin** [confident-ai/deepeval#1235]
**Is your feature request related to a problem? Please describe.**

When using DeepEval in a Git pre-push hook, the output includes all test results. This extensive output makes it difficult to quickly identify failed tests, hindering the efficiency of the pre-push process.

**Describe the solution you’d like**

I would like DeepEval to offer an option or flag that allows displaying only failed tests in the output. 

**Describe alternatives you’ve considered**

Develop a custom script using Python API to evaluate LLM outputs, bypassing the DeepEval CLI.

**Additional context**

Implementing this feature would be particularly beneficial for devs who integrate DeepEval into CI pipelines or automated Git hooks, where concise and relevant output is crucial.

**Comment by stevermeister** [comment]
I also expected it to be a built-in feature

**Comment by realei** [comment]
I totally agree! This feature would be very useful, and I’m looking forward to seeing it implemented.

**Comment by yawningphantom** [comment]
@realei @stevermeister created a PR for this https://github.com/confident-ai/deepeval/pull/2829
